src/api/assignments.ts

```diff
@@ -4,38 +4,95 @@ import { requireAuthentication } from "../util/authentication.js";
 import { Course } from "../models/courses.js";
 import { User } from "../models/users.js";
 import { Assignment } from "../models/assignments.js";
+import { Submission } from "../models/submissions.js";
 
 const router = Router();
 
-router.post("/:id/submissions", (req, res) => {
+router.post("/:id/submissions", requireAuthentication({
```

Comment:
This route should also require certain body inputs. It should also save the incoming file to disk or cloud storage.